OKTA-567519 Configurable Rate Limits for OAuth 2.0 Apps (#3900)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

##### Bar graph

The bar graph shows either the top 10 IP addresses, top 10 API tokens, or top 10 OAuth 2.0 applications that are consuming the most traffic for the API. This view enables you to check if there are IP addresses, API tokens, or applications that are creating the spike in traffic.

Display the desired bar graph by selecting **Top Requests by IP**, **Top Requests by API Token**, or **Top Requests by Application** in the dropdown menu.

> Click an IP address, API token, or Application bar once to view the traffic for that IP address, API token, or Application on the line graph.

Reviewer: ```suggestion
Click an IP address, API token, or Application bar to view the associated traffic on the line graph.
```
